Packs/Threat_Crowd/Integrations/ThreatCrowd_v2/ThreatCrowd_v2_test.py

```diff
@@ -110,6 +110,9 @@ def test_ip_command(mocker):
     assert res[0].indicator.dbot_score.reliability == DBotScoreReliability.C
     assert res[0].indicator.dbot_score.score == 3
     assert res[0].indicator.dbot_score.indicator == "0.0.0.0"
+    assert len(res[0].outputs['resolutions']) == 2
```

Comment:
and `len(res[1].outputs['resolutions']) == 0`?
please add that as well to make sure that an ip without references is kept treated properly..